ui/src/main/webapp/app/settings/settings-service.js

```diff
@@ -20,6 +20,7 @@ angular.module('app').factory('settings', function() {
     var settings = {};
 
     var resultLimit = 1000;
+    var clearChainCheckboxState = false;
```

Comment:
Just as a general thing. A service should not care about the form control used to define it. If we swapped to a switch, for example, the service variable would have to be changed. Using a variable more like `var clearChainAfterExecution = false` would be better. I think the initial value should be set to true.